test/unit/test_http_utils.py

```diff
@@ -123,3 +125,19 @@ def server(client, _):
                 self.assertEqual(''.join(rest), data[off:])
         finally:
             server_thread.kill()
+
+
+class TestSomewhatBufferedHTTPConnection(unittest.TestCase):
+
+    def test_discard_buffering_arg_on_py26(self):
```

Comment:
I'd suggest adding some empty newlines in the code below to separate code that sets up values to test assertions on from actual assertions.